docs/pages/2020_BIGMAP/sections/basics.rst

```diff
@@ -236,18 +269,14 @@ To run a ``CalcJob``, you need to set up two things: a ``code`` that is going to
 
 If you're running this tutorial in the Quantum Mobile VM or on Binder, these have been pre-configured for you. If you're running on your own machine, you can follow the instructions in the panel below.
 
-.. seealso::
-
-   More details for how to :ref:`run external codes <how-to:run-codes>`.
-
 .. dropdown:: Install localhost computer and code
```

Comment:
`localhost` will already be available, so we'll use that. Remove the dropdown and add the line that sets up the `add@localhost` code.

Make sure to fix the rest of the text/code snippets to adjust for this.